src/Classes/SkillsTab.lua

```diff
@@ -645,6 +645,23 @@ function SkillsTabClass:CreateGemSlot(index)
 		self:AddUndoState()
 		self.build.buildFlag = true
 	end)
+	slot.quality.tooltipFunc = function(tooltip)
+		if tooltip:CheckForUpdate(self.build.outputRevision, self.displayGroup) then
+			if self.displayGroup.gemList[index] then
+				local calcFunc, calcBase = self.build.calcsTab:GetMiscCalculator(self.build)
+				if calcFunc then
+					local storedQuality = self.displayGroup.gemList[index].quality
+					self.displayGroup.gemList[index].quality = 20;
```

Comment:
```suggestion
					self.displayGroup.gemList[index].quality = 20
```
No semicolons needed.